packages/common-ethereum/src/project/versioned/v1_0_0/model.ts

```diff
@@ -117,7 +121,7 @@ export class ProjectNetworkV1_0_0 extends ProjectNetworkDeploymentV1_0_0 {
   dictionary?: string;
 }
 
-export class DeploymentV1_0_0 {
```

Comment:
line 125, method is deprecated, try to use `plainToInstance`. Align with substrate